tests/app/token.tests.ts

```diff
@@ -29,24 +28,19 @@ describe('App store token tests', () => {
       env: Duda.Envs.direct,
     })
 
-    scope = nock(`https://${Duda.Envs.direct}`, {
+    scope = nock('https://api.duda.co', {
       reqheaders: {
-        'authorization': `Basic ${encoded_auth}`,
+        'x-duda-access-token': `Bearer ${token}`
```

Comment:
Only the basic auth is needed for this request, not the x-duda-access-token. Can revert.